### ISSUE TITLE
File naming

### ISSUE THREAD

**Issue by hozuki** [joaquinvanschoren/ML-course#2]
The names of neural networks slides contain `|`. This seems allowed on macOS, but it is an invalid character in Windows and Linux file names. It prevents Windows and Linux users cloning or updating this repository.

```plain
Pulling from upstreams...
remote: Counting objects: 177, done.
remote: Compressing objects: 100% (10/10), done.
remote: Total 177 (delta 4), reused 1 (delta 1), pack-reused 166
Receiving objects: 100% (177/177), 11.75 MiB | 582.00 KiB/s, done.
Resolving deltas: 100% (10/10), completed with 1 local object.
From https://github.com/joaquinvanschoren/ML-course
 * branch            master     -> FETCH_HEAD
   66e8297..3162f05  master     -> upstream/master
error: unable to create file N1 | Introduction.ipynb: Invalid argument
error: unable to create file N2 | Artificial Neuron.ipynb: Invalid argument
error: unable to create file N3 | Perceptron Classifier.ipynb: Invalid argument
error: unable to create file N4 | MLP.ipynb: Invalid argument
error: unable to create file N5 | MLP Classifier.ipynb: Invalid argument
error: unable to create file N6 | Optimization and Regularization.ipynb: Invalid argument
error: unable to create file N7 | Convolutional Networks.ipynb: Invalid argument
Updating 66e8297..3162f05
error: unable to create file N8 | Recurrent Networks.ipynb: Invalid argument
```

**Comment by hozuki** [comment]
Seems fixed in c19ab817bd5e5bdeea686b0a810446181d23cc74.

**Comment by joaquinvanschoren** [comment]
Thanks for posting the issue!
